src/main/zabbix/etc/zabbix/scripts/sonicmq_discovery.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+set -e
+
+COMPONENT=$1
+
+cd $(dirname $0)
+
+# Load environment
+. ./sonicmq_env.sh
+
+
+
+function sonicmq_get_stat {
+  if [ "$COMPONENT" = "ConnectionAggregate" ]; then
+    AGGREGATED=$(cat $SMQ_MON_OUTPUT_FILE | jq '.discovery.Connection.data | map(del(.["{#ID}"])) | map(del(.["{#NAME}"])) | unique // empty')
```

Comment:
Please add a simple comment here telling what the map commands do - delete duplicate id and name pairs?

Reply:
Actually it should probably filter out also #BROKER.